Dockerfile

```diff
@@ -44,9 +50,29 @@ ARG telemetryServer=telemetry-server
 #
 FROM ${POSTGRES_BASE}:${POSTGRES_MAJOR} AS telemetry-postgres
 
-# add the telemetry init scripting
+#
+# Build the customised telemetry-tools image
+#
+FROM ${TOOLS_BASE}:${TOOLS_MAJOR} AS telemetry-tools
+
+# Install required tools: AWS CLI, Postgres CLI, compression utilities
+# This image is used for all telemetry tooling
+# Add packages and scripts as needed
+RUN set -euo pipefail; \
+    zypper -n refresh && \
+    zypper -n install --no-recommends \
+        aws-cli \
+        postgresql16 \
```

Comment:
The 16 here probably should be the ${POSTGRES_MAJOR} value instead.

To use it though you will need to add an `ARG POSTGRES_MAJOR` line after the FROM line in this block to ensure the parameter value is accessible.

And if there is another instance of 16 used anywhere for the postgres major version, we probably should do the same there...